tools/rna_tools/compalignp/tool_dependencies.xml

```diff
@@ -0,0 +1,21 @@
+<tool_dependency>
+    <package name="squid" version="1.9g">
+        <repository name="package_squid_1_9g" owner="iuc" prior_installation_required="True" />
+    </package>
+    <package name="compalignp" version="1.0">
+        <install version="1.0">
+            <actions>
+                <action type="download_by_url">http://www.biophys.uni-duesseldorf.de/bralibase/compalignp.tgz</action>
+                <action type="set_environment_for_install">
+                    <repository name="package_squid_1_9g" owner="iuc">
+                        <package name="squid" version="1.9g"/>
+                    </repository>
+                </action>
+                <action type="shell_command">make</action>
```

Comment:
I'm just guessing here, but make is only compiling the program you still need to copy the binary into $INSTALL_DIR isn't it?